content/building/environment-variables.md

```diff
@@ -70,6 +71,39 @@ Here you'll find some of the read-only environment variables explained.
+| ANDROID_NDK_HOME               | Absolute path to Android NDK                                                         |
+| ANDROID_NDK_ROOT               | Absolute path to Android NDK                                                         |
+| ANDROID_PREFS_ROOT             | Absolute path to Android SDK and tools                                               |
+| ANDROID_SDK_ROOT               | Absolute path to Android SDK and tools                                               |
+| FLUTTER_ROOT                   | Absolute path to Flutter SDK                                                         |
+| GIT_MERGE_AUTOEDIT             | ?                                                                                    |
+| GOOGLE_APPLICATION_CREDENTIALS | ?                                                                                    |
+| GRADLE_HOME                    | Absolute path to Gradle home directory                                               |
+| HOME                           | Absolute path to build root folder                                                   |
+| JAVA_HOME                      | Absolute path to Java home directory                                                 |
+| LANG                           | ?                                                                                    |
+| LANGUAGE                       | ?                                                                                    |
+| LC_ALL                         | ?                                                                                    |
+| LC_CTYPE                       | ?                                                                                    |
+| NDK_HOME                       | Absolute path to Android NDK                                                         |
+| RBENV_VERSION                  | Ruby version                                                                         |
+| SNAPCRAFT_BUILD_ENVIRONMENT    | Snap Craft build environment                                                         |
+| SHELL                          | Absolute path to currently running shell program                                     |
+| SSH_AGENT_PID                  | ?                                                                                    |
+| SSH_AUTH_SOCK                  | ?                                                                                    |
+| SSH_CLIENT                     | ?                                                                                    |
+| SSH_CONNECTION                 | ?                                                                                    |
+| TMPDIR                         | Absolute path to dir to use for temporary files                                      |
+| USER                           | System user                                                                          |
+
```

Comment:
@BondarenkoStas can you comment on each of the `?` marks here? Either with the missing description or if we should not display it. I'm particularly not sure about the SSH_, LC, and LANG env vars.